backend/app/adapter/db/url.go

```diff
@@ -98,28 +99,42 @@ WHERE "%s"=$1;`,
 
 // GetByAliases finds URLs for a list of aliases
 func (u URLSql) GetByAliases(aliases []string) ([]entity.URL, error) {
+	// create a string for aliases parameters
+	// it looks like: $1, $2, $3, ......
+	params := make([]string, 0, len(aliases))
+	for i := range aliases {
+		params = append(params, fmt.Sprintf("$%d", i+1))
+	}
+	parameterStr := strings.Join(params, ", ")
+
+	// create a list of interface{} to hold aliases for db.Query()
+	aliasesInterface := []interface{}{}
+	for _, alias := range aliases {
+		aliasesInterface = append(aliasesInterface, alias)
+	}
```

Comment:
I couldn't find whether did you `tx.Prepare`.

Reply:
I just added prepare function in the latest commit